src/components/table/README.md

```diff
@@ -69,6 +69,7 @@ these names):
 | `_cellVariants` | Object | Bootstrap contextual state applied to individual cells. Keyed by field (Supported values: `active`, `success`, `info`, `warning`, `danger`). These variants map to classes `table-${variant}` or `bg-${variant}` (when the `dark` prop is set).
 | `_rowVariant` | String | Bootstrap contextual state applied to the entire row (Supported values: `active`, `success`, `info`, `warning`, `danger`). These variants map to classes `table-${variant}` or `bg-${variant}` (when the `dark` prop is set)
 | `_showDetails` | Boolean | Used to trigger the display of the `row-details` scoped slot. See section [Row details support](#row-details-support) below for additional information
+| `_selected` | Boolean | UI triggered. See section [Selectable rows](#row-select-support) below for additional information
```

Comment:
Is this actually used?  I don't see it referenced in the updated code.

Reply:
you're right, this is from a previous version and not used anymore. I need to fix this